File: data/java/FIND_INDEX_OF_AN_EXTRA_ELEMENT_PRESENT_IN_ONE_SORTED_ARRAY_1_TRANSLATED.py
====================
def f_gold ( arr1 , arr2 , n ) :
    index = n
    left , right = 0 , n - 1
    while left <= right :
        mid = ( left + right ) // 2
        if arr2 [ mid ] == arr1 [ mid ] :
            left = mid + 1
        else :
            index = mid
            right = mid - 1
    return index

File: data/python/FIND_INDEX_OF_AN_EXTRA_ELEMENT_PRESENT_IN_ONE_SORTED_ARRAY_1_TRANSLATED.java
====================
public static int f_gold ( int [ ] arr1 , int [ ] arr2 , int n ) {
  int index = n ;
  int left = 0 ;
  int right = n - 1 ;
  while ( ( left <= right ) && ( arr2 [ left ] == arr1 [ right ] ) ) {
    int mid = ( int ) ( ( left + right ) / 2 ) ;
    if ( ( arr2 [ mid ] == arr1 [ mid ] ) && ( arr1 [ mid ] == arr2 [ mid ] ) ) {
      left = mid + 1 ;
    }
    else {
      index = mid ;
      right = mid - 1 ;
    }
  }
  return index ;
}

